docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: your title
+permalink: /docs/rfcs/file-name
+---
+
+* RFC PR: [PR #](https://github.com/mozilla-mobile/android-components/pull/#)
+* Start date: YYYY-MM-DD (Day of proposal posting.)
+* End date: YYYY-MM-DD (Last day for general feedback. However, the proposal can be merged immediately after all stakeholders approve.)
+* Stakeholders: github-username, github-username
+
+## Summary
+
+This section should include a brief description of the proposal.
+
+## Motivation
+
+This section should include reasoning about why the proposal is useful. Examples, specific scenarios, 
+open bugs, records of performance metrics, and other empirical data are beneficial to include here if available.
+
+## Guide-level explanation
+
+This section should include a high-level walkthrough of the steps required to implement the proposal.
+
+## Reference-level explanation (optional)
+
+This section should include a detailed walkthrough of technical steps or code changes that 
+will be required to implement the proposal. Code samples and prototypes are beneficial to include here.
+
+## Drawbacks (optional)
+
+This section should include any drawbacks to the proposal.
+
+## Rationale and alternatives (optional)
```

Comment:
```suggestion
## Rationale and alternatives
```

Sorry, I missed this in my previous pass. For the same reason as the comment above on 'Unresolved questions', let's make it required that the developer explicitly says there are no alternatives to their design.